plugins/telemetry-otel/src/main/java/org/opensearch/telemetry/tracing/exporter/OtlpGrpcSpanExporterProvider.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.telemetry.tracing.exporter;
+
+import org.opensearch.common.settings.Settings;
+import org.opensearch.telemetry.OTelTelemetrySettings;
+
+import java.util.Collection;
+
+import io.opentelemetry.exporter.otlp.trace.OtlpGrpcSpanExporter;
+import io.opentelemetry.sdk.common.CompletableResultCode;
+import io.opentelemetry.sdk.trace.data.SpanData;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+
+/**
+* OtlpGrpcSpanExporterProvider class creates an instance of OtlpGrpcSpanExporter
+*/
+public final class OtlpGrpcSpanExporterProvider implements SpanExporter {
+
+    private OtlpGrpcSpanExporter delegate;
```

Comment:
The `OtlpGrpcSpanExporter` under the hood uses service discovery to find out suitable `GrpcSenderProvider`, it seems like you are inclined to use `OkHttpGrpcSenderProvider` but I don't see this dependency anywhere, what sender provider is being used?